src/Security/Authentication/JwtBearer/samples/MinimalJwtBearerSample/appsettings.Development.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "Authentication": {
```

Comment:
Should this have a `"DefaultScheme"` profile too? At the moment this sample doesn't have a default schema configured.